Applications/Utils/FileConverter/NetCdfConverter.cpp

```diff
@@ -566,9 +588,22 @@ static bool convert(NcFile const& dataset, NcVar const& var,
             if (i == time_bounds.second)
             {
                 MeshLib::IO::VtuInterface vtu(mesh.get());
-                vtu.writeToFile(output_name);
+                std::string const output_file_name =
+                    (BaseLib::getFileExtension(output_name) == ".vtu")
+                        ? "" : output_name + ".vtu";
+                vtu.writeToFile(output_file_name);
             }
         }
+        else //OutputType::IMAGES
+        {
+            GeoLib::Raster raster(
```

Comment:
tiny: this probably too, could be const.

Reply:
✔️